proto/xmtpv4/message_api/message_api.proto

```diff
@@ -83,37 +68,60 @@ message GetInboxIdsResponse {
   repeated Response responses = 1;
 }
 
-// Replication API
+message SubmitMisbehaviorReportRequest {
+  xmtp.xmtpv4.envelopes.UnsignedMisbehaviorReport report = 1;
+}
+
+message SubmitMisbehaviorReportResponse {}
+
+message QueryMisbehaviorReportsRequest {
+  uint64 after_ns = 1;
```

Comment:
I think this is the minimum set of parameters we should be able to query by, but in the future it may make sense to add queries by `misbehaving_node_id` or misbehavior type or queries that exclude client reports